index.js

```diff
@@ -141,6 +144,12 @@ File.prototype.inspect = function() {
   return '<File ' + inspect.join(' ') + '>';
 };
 
+File.builtInFields = ['_contents', 'contents', 'stat', 'history', 'path', 'base', 'cwd'];
```

Comment:
I'm open to suggestions. My main concern is for it to be configurable on objects that extend from `Vinyl`. With this implementation you can just do:

```js
class Vinyl2 extends Vinyl {}
Vinyl2.builtInFields = ['foo', 'bar', ...Vinyl2.builtInFields];
```

And everything works as expected.